### ISSUE TITLE
[Linux] OpenGL finding problem

### ISSUE THREAD

**Issue by Kron4ek** [LWJGL/lwjgl#128]
2.9.2 and 2.9.3 versions of LWJGL have some problem with opengl finding. Because of this, does not run Minecraft (no opengl context found in the current thread). But the version 2.9.1 has no such problem.

I'm using Debian 8 Jessie and Nvidia 340.96

**Comment by Spasi** [comment]
Do you have Mesa installed in your system? If yes, does 2.9.2+ work if you uninstall Mesa?

**Comment by Kron4ek** [comment]
No, Mesa is not installed.

**Comment by Spasi** [comment]
I went through the change log from 2.9.1 to 2.9.2 and couldn't find anything that might be responsible. Most Linux-related changes had to do with Xrandr.

Would you be able to clone LWJGL, build it locally and try one of the OpenGL demos?

**Comment by Kron4ek** [comment]
I found a solution. It was necessary to install xrandr (in Debian is the package x11-xerver-utils). So, versions 2.9.2+ require xrandr and older versions do not require it.

**Comment by ishitatsuyuki** [comment]
Yes, it depends on Xrandr or throws out-of-bound. I have confirmed this and I'd like to bump this for a fix.
